spring-integration-core/src/main/java/org/springframework/integration/config/SpelFunctionFactoryBean.java

```diff
@@ -42,9 +44,9 @@ public class SpelFunctionFactoryBean implements FactoryBean<Method>, Initializin
 
 	private final String functionMethodSignature;
 
-	private String functionName;
+	private @Nullable String functionName;
 
-	private Method method;
+	private @Nullable Method method;
```

Comment:
The `@SuppressWarnings("NullAway.Init")` on both properties.
One of them is from `setBeanName()` and another from the `afterPropertiesSet()`.
I doubt we should pursue the case when this class is not used as a bean in the code.